web/src/components/SingleRoom.tsx

```diff
@@ -260,7 +68,141 @@ const SingleRoom: React.FC<Props> = ({ roomId, userId }) => {
           micOn={micOn}
           speakerOn={speakerOn}
         />
-        <MomentaryChat roomId={roomId} userId={userId} nickname={nickname} />
+        <div>
+          <UserProfile
+            initialNickname={initialNickname}
+            emoji={emoji}
+            onUpdateNickname={(text) => {
+              setNickname(text);
+              setStringItem("nickname", text);
+            }}
+            onUpdateStatusMesg={(text) => {
+              setStatusMesg(text);
+            }}
+            onUpdateEmoji={(e) => {
+              setEmoji(e);
+            }}
+          />
+          <button
+            type="button"
+            className="SingleRoom-config-toggle"
+            onClick={() => setConfigOpen((o) => !o)}
+          >
+            Setting{configOpen ? <>&#9660;</> : <>&#9654;</>}
```

Comment:
I'm not sure if triangles are good UI in this case. We can revisit this later with bigger pictures.